OpenRA.Mods.Common/Traits/World/ActorMap.cs

```diff
@@ -515,7 +526,7 @@ public void UpdateProximityTrigger(int id, WPos newPos, WDist newRange, WDist ne
 
 		public void AddPosition(Actor a, IOccupySpace ios)
 		{
-			UpdatePosition(a, ios);
```

Comment:
Are you sure that this change is safe?
This line made sure to clean up the previous position if the caller forgot to `RemovePosition` first.

Reply:
We could remove these changes from this PR, but when i debugged this it looked like this was not needed. If i remember correctly.